<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyWindow</class>
 <widget class="QWidget" name="ProxyWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxy - SecuLite</string>
  </property>
  
  <!-- Proxy Settings Section -->
  <widget class="QGroupBox" name="proxy_settings_group">
    <property name="title">
      <string>Proxy Settings</string>
    </property>
    <property name="geometry">
      <rect>
        <x>10</x>
        <y>10</y>
        <width>350</width>
        <height>150</height>
      </rect>
    </property>
    
    <widget class="QLineEdit" name="host_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Proxy Host</string>
     </property>
    </widget>
    
    <widget class="QLineEdit" name="port_input">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Proxy Port</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="test_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Proxy</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="toggle_intercept_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle Intercept</string>
     </property>
    </widget>
  </widget>
  
  <!-- Intercepted Requests Section -->
  <widget class="QGroupBox" name="intercepted_requests_group">
    <property name="title">
      <string>Intercepted Requests</string>
    </property>
    <property name="geometry">
      <rect>
        <x>10</x>
        <y>170</y>
        <width>780</width>
        <height>300</height>
      </rect>
    </property>
    
    <widget class="QListWidget" name="intercepted_requests_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>760</width>
       <height>230</height>
      </rect>
     </property>
    </widget>
    
    <widget class="QPushButton" name="forward_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="drop_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Drop</string>
     </property>
    </widget>
    
    <widget class="QPushButton" name="to_repeater_button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Send to Repeater</string>
     </property>
    </widget>

    <widget class="QPushButton" name="to_intruder_button">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Send to Intruder</string>
     </property>
    </widget>
  </widget>
  
  <!-- Logs -->
  <widget class="QLabel" name="log_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>760</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Logs will appear here...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
